### DISCUSSION TITLE
Using mdbook-i18n-helpers on Windows?

### DISCUSSION THREAD

**Discussion by savbran** [google/mdbook-i18n-helpers#23] [General]
I have rust toolchain installed on Windows and the necessary crates (mdbook and this helpers). I tried to use it on Windows but there are strange messages indicating gettext is not installed nevertheless it is.

Did you tried it on Windows?

Thanks inadvance.

![image](https://user-images.githubusercontent.com/3647689/234020143-307c6aaa-5e8e-45ee-bca6-0ce5d747d0da.png)

**Comment by mgeisler** [comment]
Hi @savbran, I'm sorry for not seeing this sooner! I somehow hadn't enabled notifications for the repository.

I have not tried this on Windows, but I think I can help using the helpful screenshot above.

What is missing is not a `gettext` binary — the plugins here don't actually depend on Gettext (though you still want the Gettext tools installed so that you can run `msgmerge` and other Gettext utilities).

The missing commands are `mdbook-gettext` and `mdbook-xgettext`. I assume you ran either

- `cargo install mdbook-i18n-helpers` or
- `cargo install --path .` after cloning the repository?

At the end of the installation output, you should see where the binaries are stored. On Linux, it looks like this:

```
cargo install mdbook-i18n-helpers
[...]
   Compiling handlebars v4.3.7
   Compiling mdbook v0.4.34
   Compiling mdbook-i18n-helpers v0.2.2
    Finished release [optimized] target(s) in 10.76s
  Installing /home/mg/.cargo/bin/mdbook-i18n-normalize
   Replacing /home/mg/.cargo/bin/mdbook-gettext
   Replacing /home/mg/.cargo/bin/mdbook-xgettext
```

The next step is to add the path to the `PATH` environment variable. From https://www.rust-lang.org/tools/install, it seems that this should happen automatically during the Rust install. Perhaps it has already been done then?
